### DISCUSSION TITLE
AMA

### DISCUSSION THREAD

**Discussion by upintheairsheep** [FunkinCrew/Funkin#1961] [Q&A] [answered]
What phone model do gf, bf, dd, mm, and pico use currently?
Is it possible to get gf’s heart from bf?
What is gf’s, bf’s, dad’s, mom’s, and pico’s favorite social media other than Newgrounds?
Girlfriend’s last name is Dearest and Boyfriend’s is Hatsune. Am I correct on the 2?
What will happen if somebody r-pes gf? What will she do to him? Kill him? Eat him like how a Lion eats an anotolpe? Put him straight to hell?
Are the Backrooms and/or the SCP Foundation canon to the universe?
Where does the game take place?
Will you port the game to smart tv?
What would be your reaction if for some reason, Girlfriend quantum tunneled onto real earth?
What is GF’s height, width, and weight in your preferred unit?
Does GF use Windows 10 or 11 (or something else?)

**Comment by conecat** [comment, marked as answer]
interesting questions lol

CAMERON HOW IS THIS AN ANSWER LOLLLLLL
